k8s-dev/cstor-pool-mgmt/pool-resource-cluster-1.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: openebs.io/v1alpha1
+kind: CStorPool
+metadata:
+  name: pool1-abc
+  node: node-host-label
+  labels:
+#The following label will be used for additional node reference, since we go for UID based selection of pod.
+    "kubernetes.io/hostname": "node-host-label"
+
+#finalizer helps handling deletion of resource
+  finalizers: ["cstorpool.openebs.io/finalizer"]
+
+spec:
+  disks:
+   diskList: ["/dev/sdc1"]
+  poolSpec:
+   cacheFile: /tmp/pool1.cache
+   poolType: striped
+   overProvisioning: false
+status:
```

Comment:
A status is never set in a yaml. It should be set by the controller even if its a `init`.

Reply:
As discussed, this will be handled in separate PR and will be enhanced when k8s sub resource is grown, which is right now in initial phase.